packages/date-picker/docs/Calendar.stories.tsx

```diff
@@ -97,6 +97,12 @@ const StickerSheetCalendarTemplate: Story<{ isReversed: boolean }> = ({
           id="today-selected"
           value={new Date("2022-05-01")}
         />
+        <CalendarExample
```

Comment:
Should we actually add all the combination hover+focus states to the sticker sheet too to capture any possible unintentional overridden styles? Or would it be overkill?